### ISSUE TITLE
Initialising with a filter

### ISSUE THREAD

**Issue by virtualLast** [patrickkunka/mixitup#316]
I have the following code
`
$container.mixItUp({
				load: {
					filter: '.body_1'
				}
			});
`
but the items in my list are not being filtered when the page loads. All the code is wrapped in a document.ready function, filtering is done via the data api - any idea's?

**Comment by patrickkunka** [comment]
It looks like you are using MixItUp v2 (or at least the v2 jQuery API). If you could upgrade to v3 I'd be happy to take a look at this.

**Comment by virtualLast** [comment]
what do you mean? I have included v3.1.6

**Comment by patrickkunka** [comment]
OK - great. If you are using the v3 API your instantiation code should look like this:

```js
var mixer = mixitup($container, {
    load: {
        filter: '.body_1'
    }
});
```

That alone may sort out your issues, but if not, it would be great if you could include a link to a staging site or reproduction of the issue.
